src/GitHub.VisualStudio/UI/Views/Controls/LoginControl.xaml.cs

```diff
@@ -39,7 +43,10 @@ public LoginControl()
                         NotifyDone();
                     }
                 });
+
+                d(Disposable.Create(() => errorHandler.Dispose()));
```

Comment:
I don't think so - a tab will be selected when the UI is built, causing the selection changed event to fire. Looking at the WPF call stack it looks like this will always happen.